app.js

```diff
@@ -36,6 +38,15 @@ app.use('/api/fleets', fleetsRouter);
 app.use(express.static("public"))
 
 
+//CORS settings
+app.get('/products/:id', function (req, res, next) {
```

Comment:
Did you mean anything else but products?

Reply:
oh, ty! sorry didn't realise since it worked just fine